addons/mail/static/src/owl/components/chatter/chatter.js

```diff
@@ -23,43 +28,80 @@ class Chatter extends Component {
                 id: props.id,
             });
             return {
+                composerLocalId: thread ? thread.composerLocalId : undefined,
                 threadLocalId: thread ? thread.localId : undefined,
             };
         });
         this._threadRef = useRef('thread');
     }
 
-    mounted() {
-        // TODO {xdu}
-        // Need to say to the underlying Thread that there is no long polling
-        // So when doing message_post, you need to do a read on the record and
-        // then verify in the message_ids if there are missing messages or not
-        // and then make a message_format call for each message (including the
-        // one posted)
+    async willStart() {
+        await this.storeDispatch('initChatter', {
+            model: this.props.model,
+            id: this.props.id,
+        });
```

Comment:
Chatter component should only work on an existing "record thread", so it must be created in form renderer in case it doesn't exist yet

Reply:
You mean this dispatch should be done in `form_renderer.js` ?